### DISCUSSION TITLE
Idea: More/deeper content for each ambassador

### DISCUSSION THREAD

**Discussion by pjeweb** [alveusgg/extension#19] [Ideas]
1. Include a slider or something similar with more pictures for each ambassador. The pics from the website could be a starting point.
2. Add links (or embeds) to ambassadors to/of video content (intro stream, animal quest, instagram reel animal quest, training sessions, timestamped highlights on educational collabs etc.)
3. Link the appropriate IUCN Redlist page if evaluated (eg. Mia: https://www.iucnredlist.org/species/22724813/154428817)

**Comment by pjeweb** [comment]
Oh one idea was also to add the Facts or FAQ from the OBS overlays to the extension.

**Reply by abdullahmorrison** [reply]
Would depend on the design. would need to see a wireframe of designs on all sizes of display.

**Comment by abdullahmorrison** [comment]
1. This will be difficult to make responsive to different displays (including mobile). would need to see it designed on all sizes
2. I plan on adding a link to each ambassador's animal quest. Other stuff would cause too much clutter.
3. I think the only good way to do this would be to add a link inside the ambassador card in the IUCN status part of the card. The problem is that it would make the card look ugly if it were a visible link. If it were a subtle link it would be too hard to find. Open to ideas.
